### PR TITLE
create thankyou.sustain.hbs

### PR DESCRIPTION
**Pull request by piamancini**
we need to make a thankyou for the sustain donors so they now they are confirmed for the event.

### CODE REVIEW

File: templates/emails/thankyou.sustain.hbs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <li>You can follow our activities and how we will be spending the money on our open collective page:
+  <a href="https://opencollective.com/{{group.slug}}">https://opencollective.com/{{group.slug}}</a>.</li>
+  <li>Follow us on <a href="https://twitter.com/sustainoss">Twitter</a></li>
+  <li>Open Source with us on <a href="https://github.com/sustainers">GitHub</a></li>
+  <li><a href="https://slack.opencollective.com">Slack</a> with us.</li>
+ </p>
+ 
+ <p>If you have any questions feel free to contact us:
+ Chad Whitacre: chad@gratipay.com
+ Justin Dorfman: jdorfman@tickermule.com
```
Comment (whit537) [inline templates/emails/thankyou.sustain.hbs]: tickermule → stickermule

Comment (piamancini) [inline templates/emails/thankyou.sustain.hbs]: good catch @whit537

File: templates/emails/thankyou.sustain.hbs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <p>Thank you so much for being part of the first $ustainOSS Conversation and supporting us with {{{currency donation.amount currency=donation.currency}}} per {{interval}}.
+      Your first payment was received on {{{moment donation.createdAt}}}. Your donation helps us make this conversation happen!</p>
+  {{else}}
+    <p>We would like to thank you for your continued support. Your latest {{interval}}ly donation of {{{currency donation.amount currency=donation.currency}}} was received on {{{moment transaction.createdAt}}}.
+      Your donation helps us continue our activities.</p>
+  {{/if}}
+  {{> charge_date_notice}}
+{{else}}
+<p>Thank you so much for being part of the first $ustainOSS Conversation and suppoorting us with {{{currency donation.amount currency=donation.currency}}} on {{{moment donation.createdAt}}}.
+  If you made a $50 donation for a ticket, we look forward to meeting you! We will be emailing details shorlty. You do not have to worry about bringing a ticket but do save this email just in case.</p>
```
Comment (asood123) [inline templates/emails/thankyou.sustain.hbs]: shortly misspelled

File: templates/emails/thankyou.sustain.hbs [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  {{#if firstPayment }}
+    <p>Thank you so much for being part of the first $ustainOSS Conversation and supporting us with {{{currency donation.amount currency=donation.currency}}} per {{interval}}.
+      Your first payment was received on {{{moment donation.createdAt}}}. Your donation helps us make this conversation happen!</p>
+  {{else}}
+    <p>We would like to thank you for your continued support. Your latest {{interval}}ly donation of {{{currency donation.amount currency=donation.currency}}} was received on {{{moment transaction.createdAt}}}.
+      Your donation helps us continue our activities.</p>
+  {{/if}}
+  {{> charge_date_notice}}
+{{else}}
+<p>Thank you so much for being part of the first $ustainOSS Conversation and suppoorting us with {{{currency donation.amount currency=donation.currency}}} on {{{moment donation.createdAt}}}.
```
Comment (asood123) [inline templates/emails/thankyou.sustain.hbs]: supporting misspelled

**Comment by piamancini** [comment]
@asood123 ok, let me know what else I need.

**Pull request merged**